File: cc/spoj/FCTRL.py
#python 3 code
def z(num):
	count = 0
	for i in range(num+1):
		m = i
		while True:
			if ((m / 5 != 0) and (m % 5 == 0)):
				count = count + 1
				m = m / 5
			else:
				break
	return count				


n = int(input())
for i in range(n):
	k = int(input())
	print (z(k))

File: cc/spoj/FCTRL.cc
#include<bits/stdc++.h>
using namespace std;

int z_factorial(int num){
	int count = 0;
	while(num > 4){
		num /= 5;
		count += num;
	}
	return count;
}
int main(int argc, char const *argv[])
{
	
	int n,k;
	cin>>n;
	
	// cout<<mod(n,k);return 0;
	
	for (int i = 0; i < n; ++i)
	{
		cin>>k;
		cout<<z_factorial(k)<<endl;
	}
	
	return 0;
}
